tools/run_tests/run_interop_tests.py

```diff
@@ -769,6 +831,9 @@ def aggregate_http2_results(stdout):
                   action='store_const',
                   const=True,
                   help='Run cloud_to_prod_auth tests.')
+argp.add_argument('--gcr_tag',
```

Comment:
I'd prefer something very minimal like "--keep_docker_images" that would avoid removing docker images once the script finishes and printed/saved the list of image names kept.
With that input, you can then have a separate script that does the rest.